.travis.yml

```diff
@@ -1,12 +1,62 @@
 language: dart
-sudo: false
 dart:
-  - stable
   - dev
+
 install:
   - gem install coveralls-lcov
-script: ./tool/travis.sh
+
+# Gives more resources on Travis (8GB Ram, 2 CPUs).
+# Do not remove without verifying w/ Travis.
+sudo: required
+addons:
+  chrome: stable
```

Comment:
chrome is only required for browser tests (and if the tests are also done on chrome)

Reply:
Yes, there are browser tests, hence why we are installing Chrome.